crates/runtime/benches/bench.rs

```diff
@@ -7,7 +7,7 @@
 
 // spice.ai/spicehq/spice-tests/datasets/spicehq."spice-tests".oss_benchmarks
 // schema
-// run_id, started_at, finished_at, query_name, status, min_duration, max_duration, iterations, commit_sha
```

Comment:
The benchmark run failed: https://github.com/spiceai/spiceai/actions/runs/10040651623/job/27747115371